gluster/block/Makefile

```diff
@@ -0,0 +1,69 @@
+# Copyright 2017 The Kubernetes Authors.
```

Comment:
This is the only of the 4 commits in this PR that actually does what the PR title says: add a makefile.
I guess the other commits should be separate PRs?